services/movie_session.py

```diff
@@ -0,0 +1,46 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    return (
+        MovieSession.objects.filter(show_time__date=session_date)
+        if session_date
+        else MovieSession.objects.all()
+    )
```

Comment:
It would be better to write this code in multiple lines and have a simple `return` statement to improve the readability.